Submitted By: Mario Fetka (mario dot fetka at gmail dot com)
Date: 2008-09-02
Initial Package Version: 0.9.2
Origin: me
Upstream Status: unknown
Description: Make it compile with gcc 4.3
 
diff -Naur VendorSupplied.orig/Epson.PDC/OmniEpsonBlitter.cpp VendorSupplied/Epson.PDC/OmniEpsonBlitter.cpp
--- VendorSupplied.orig/Epson.PDC/OmniEpsonBlitter.cpp	2004-06-09 14:12:53.000000000 +0000
+++ VendorSupplied/Epson.PDC/OmniEpsonBlitter.cpp	2008-09-02 13:24:03.118003224 +0000
@@ -111,6 +111,9 @@
 #include "Omni.hpp"
 #include "JobProperties.hpp"
 
+#include <cstring>
+#include <cstdlib>
+
 /****************************************************************************/
 /* Epson changable...                                                       */
 /****************************************************************************/
